components/automate-deployment/pkg/backup/spec.go

```diff
@@ -303,6 +303,12 @@ func DefaultSpecs(serviceNames []string) []Spec {
 					SrcPath: "/hab/svc/ingest-service/data",
 				},
 			},
+			SyncDbsV2: []DatabaseDumpOperationV2{
+				{
+					Name: "chef_ingest_service",
```

Comment:
i had a dream about this being annoying because we would have to specify the tables. i'm glad that was just a nightmare